torch/__init__.py

```diff
@@ -2587,6 +2587,27 @@ def get_device_module(device: _Optional[_Union[torch.device, str]] = None):
     return device_module
 
 
+def cumsum(
```

Comment:
I'm not sure exactly what is the right way to override the Python `torch.cumsum` function. This way causes a few mypy failures

Reply:
I realized that I need to put this in `torch/functional.py`